test/Dapr.Client.Test/SecretApiTest.cs

```diff
@@ -93,6 +95,32 @@ public async Task GetSecretAsync_ReturnSingleSecret()
             secretsResponse["redis_secret"].Should().Be("Guess_Redis");
         }
 
+        [Fact]
+        public async Task GetSecretAsync_WithSlashesInName()
+        {
+            await using var client = TestClient.CreateForDaprClient();
+
+            var request = await client.CaptureGrpcRequestAsync(async DaprClient =>
+            {
+                return await DaprClient.GetSecretAsync("testStore", "us-west-1/org/xpto/secretabc");
+            });
+
+            request.Dismiss();
+
+            //Get Request and validate
+            var envelope = await request.GetRequestEnvelopeAsync<Autogenerated.GetSecretRequest>();
+            envelope.StoreName.Should().Be("testStore");
+            envelope.Key.Should().Be("us-west-1/org/xpto/secretabc");
```

Comment:
Would it be worth defining a method-level constant for the value of `"us-west-1/org/xpto/secretabc"` since it's repeated 5 times in this test?

Reply:
Sure, it could be. But this is just a unit test and not part of the library, so I habitually tend to favor ease of readability over performance or optimization.